pb/metainfo.proto

```diff
@@ -60,6 +60,10 @@ service Metainfo {
     // Server side move.
     rpc BeginMoveObject(ObjectBeginMoveRequest) returns (ObjectBeginMoveResponse);
     rpc FinishMoveObject(ObjectFinishMoveRequest) returns (ObjectFinishMoveResponse);
+
+    // Server side copy.
+    rpc BeginCopyObject(ObjectBeginCopyRequest) returns (ObjectBeginCopyResponse);
```

Comment:
you need to add both (BeginCopyObject/FinishCopyObject) to batch request/response

Reply:
Sorry I don't understad. What do you mean?

Reply:
each new rpc method request/response needs to be added to BatchRequestItem and BatchResponseItem https://github.com/storj/common/blob/c2e4025a226142bd2c13acea1a367287290ed3d9/pb/metainfo.proto#L672

Reply:
Done, sorry for delay.